java/src/main/java/ai/rapids/cudf/AssertEmptyNulls.java

```diff
@@ -0,0 +1,18 @@
+package ai.rapids.cudf;
+
+public class AssertEmptyNulls {
+  /*
+  We have to read the system property to see if we should by-pass this assert. Ideally we should
+  just use `-da:AssertNonEmptyNulls` as per Java standard, but the reason for doing this is because
```

Comment:
I am confused by this. surefire does not let us disable assertions at a class level?

Is argLine not enough? https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#argLine or is there something else that I am missing.  For the scala testing I think there is something similar.

Reply:
Thanks. I wasn't aware of this parameter. I was using the `enableAssertions` parameter which only supports "ea" and "da"